{{cookiecutter.project_slug}}/app/tasks/ping.py

```
@@ -0,0 +1,26 @@
+import tempfile
+from pathlib import Path
+from celery import shared_task
+from django.core.files import File
+from app.domain.background_task import attach_file_to_task, log_progress
+
+@shared_task(bind=True)
+def ping(self):
+    log_progress(self.request.id, "Sending Pong")
+    return "pong"
+
+@shared_task(bind=True)
```

Flake8 (7.2.0)

[error] expected 2 blank lines, found 1

(E302)